doc/source/installation.rst

```diff
@@ -6,18 +6,18 @@ Installation
 .. hint::
 
    This document describes how to install KIWI. Apart from the preferred
-   method to install KIWI via rpm, it is also provided on pypi and can
-   be installed via pip.
+   method to install KIWI via rpm, it is also available on `pypi
+   <https://pypi.org/project/kiwi/>`__ and can be installed via pip.
 
-KIWI can be installed with different methods. For this guide, only the
+KIWI can be installed via different methods. For this guide, only the
 installation as a packages through a package manager is described.
 
-Packages for the new KIWI version are provided at the `openSUSE
+Packages for the new KIWI version are provided by the `openSUSE
 buildservice <http://download.opensuse.org/repositories/Virtualization:/Appliances:/Builder>`__.
 
 To install KIWI, do:
 
-1. Open the URL http://download.opensuse.org/repositories/Virtualization:/Appliances:/Builder
+1. Open the URL https://download.opensuse.org/repositories/Virtualization:/Appliances:/Builder
```

Comment:
It is also packaged in Fedora and installable by just running `dnf install kiwi-cli` (technically `dnf install kiwi` works too, since the `kiwi` name is provided by `kiwi-cli`).

Reply:
I haven't found a good place where to fit that in, so I extended the installation section and added two additional sections: one for installing from pip and one to install from your distro's package manager.